frontend/src/Navigation/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { createBottomTabNavigator } from 'react-navigation-tabs';
+import Ionicons from 'react-native-vector-icons/Ionicons';
+import Settings from '../Settings';
+import Buildings from '../Buildings';
+import Colors from '../common/colors';
+
+const TabNavigator = createBottomTabNavigator({
+  'My Kingdom': {
+    screen: Buildings,
+    navigationOptions: {
+      tabBarIcon: () => <Ionicons name="ios-home" size={25} style={{ color: 'white' }} />,
```

Comment:
please add white to the `common/colors.js` and import it from there, we'll use it everywhere